raw_data/preprocessed/africa/morocco_french.txt

```diff
@@ -0,0 +1,219 @@
+Chapitre Premier - Dispositions générales 
+
+Section Première - Objectifs et principes généraux 
+
+(1) La présente loi a pour objet d'édicter les règles de base et les principes généraux de la politique nationale dans le domaine de la protection et de la mise en valeur de l'environnement. Ces règles et principes visent à: - protéger l'environnement contre toutes formes de pollution et de dégradation quelle qu'en soit l'origine; - améliorer le cadre et les conditions de vie de l'homme; - définir les orientations de base du cadre législatif, technique et financier concernant la protection et la gestion de l'environnement; - mettre en place un régime spécifique de responsabilité garantissant la réparation des dommages causés à l'environnement et l'indemnisation des victimes.
```

Comment:
You can replace all of the dashes with bullet points with the following command:

```
sed -E ':start s/^(\(1\).*)- /\1• /;t start'
```